File: Api/Entities/Frame.cs
using System;
using System.Collections.Generic;

namespace Api.Entities
{
    public partial class Frame
    {
        public Frame()
        {
            FramePictures = new HashSet<FramePicture>();
            FrameSettings = new HashSet<FrameSetting>();
            FrameUserPermissions = new HashSet<FrameUserPermission>();
            FrameUsers = new HashSet<FrameUser>();
        }

        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime CreatedDate { get; set; }
        public int CreatedBy { get; set; }
        public DateTime? EndDate { get; set; }

        public User CreatedByNavigation { get; set; }
        public ICollection<FramePicture> FramePictures { get; set; }
        public ICollection<FrameSetting> FrameSettings { get; set; }
        public ICollection<FrameUserPermission> FrameUserPermissions { get; set; }
        public ICollection<FrameUser> FrameUsers { get; set; }
    }
}

File: src/app/models/Frame.ts
import { DataTranslator } from './DataTranslator';

export class Frame extends DataTranslator {
    id: string;
    title: string;
    description: string;
    createdDate: Date;
    createdBy: string;

    constructor(id: string, title: string, description: string, createdDate: Date, createdBy: string) {
        super();
        this.id = id;
        this.title = title;
        this.description = description;
        this.createdDate = createdDate;
        this.createdBy = createdBy;
    }
}
